string name
string value
string type
---
int32 status
string message